src/NuGet.Core/NuGet.PackageManagement/PackageDownloader.cs

```diff
@@ -98,6 +99,18 @@ public static class PackageDownloader
                 groups.Enqueue(localGroup);
                 groups.Enqueue(otherGroup);
 
+                HashSet<string> nameSpacePackageSources = downloadContext.PackageNamespacesConfiguration?.Match(packageIdentity.Id);
+
+                if (nameSpacePackageSources != null)
+                {
+                    var packageSourcesAtPrefix = string.Join(", ", nameSpacePackageSources);
+                    logger.LogDebug(string.Format(CultureInfo.CurrentCulture, Strings.PackageNamespacePrefixMatchFound, packageIdentity.Id, packageSourcesAtPrefix));
+                }
+                else
+                {
+                    logger.LogDebug(string.Format(CultureInfo.CurrentCulture, Strings.PackageNamespacePrefixNoMatchFound, packageIdentity.Id));
```

Comment:
I don't know if a code analyzer suggested to you to use `CultureInfo.CurrentCultre`, but:
1. Isn't `CultureInfo.CurrentCulture` the default when you don't explicitly pass a culture to `string.Format`?
2. The string isn't formatting any numbers or dates, so culture doesn't impact this string format anyway.

Reply:
Ok. Addressed.